### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by abuehrle**
single anchor fixed

### CODE REVIEW

**Comment by abuehrle** [comment]
Updated screen captures. 

Needs revisions to text as well.

**Comment by 2opremio** [comment]
That search screenshot doesn't look right (the _cart_ text in the canvas should be highlighted)

File: site/features.md [hunk 1]
```diff
@@ -17,17 +17,17 @@ Browse the current feature set for Scope with links to relevant indepth topics:
...
-##<a name="flexible-view-filtering"></a>Flexible Filtering
+##<a name="flexible-filtering"></a>Flexible Filtering
 
 Nodes can be filtered by various properties, for example in the Container view, you can filter by System Containers vs Application Containers or by Stopped vs. Running Containers. Scope can also display various metrics such as CPU and Memory usage in the nodes, for example allowing you to easily find the container using the most CPU or memory.  Mouse-over any of the nodes to view its filtered metric at a glance.
 
 ###<a name="powerful-search"></a> Powerful Search
 
 Powerful search capability enables you to find node types, containers and processes by name, label or even path.  The search functionality supports simple operands so that you can for example, find processes consuming a certain Memory limit or nodes using too much CPU. You can also stack filters to create custom, complex search criterion. 
 
-!['Stack Filters'](images/search-python.png)
```
Comment (2opremio) [inline site/features.md]: Does this mean that the `search-python.png` image can be removed? If so, please do.

Also, please review if any other images could be removed.

**Comment by abuehrle** [comment]
@2opremio PTAL  Thinking about putting all of the search criterion on its own page as reference as well, wdyt? (expand what's in the online help for that).

File: site/features.md [hunk 2]
```diff
@@ -6,44 +6,69 @@ menu_order: 15
...
 Choose an overview of your container infrastructure, or focus on a specific microservice. Identify and correct issues to ensure the stability and performance of your containerized applications.
 
-##<a name="interact-with-and-manage-containers"></a>Interact With and Manage Containers
-
-Interact with your containers directly: pause, restart and stop containers without having to leave the Scope browser window.
+##<a name="interact-with-and-manage-containers"></a>Troubleshoot and Manage Containers
 
-##<a name="troubleshoot-apps"></a>Troubleshoot Apps
+Troubleshoot your app by directly interacting with containers: pause, restart and stop containers without having to leave the Scope browser window.
+A convenient terminal window is provided that enables you to interact with your app and to troubleshoot and diagnose any issues all within the same context.Select `<_` to launch a terminal in any container from the details panel.
```
Comment (2opremio) [inline site/features.md]: There's many terminals,  `<_` should be `>_`

**Comment by 2opremio** [comment]
@abuehrle You added a full refactoring of the docs, piggybacking on a PR supposed to fix an anchor. That's not a good practice, since it makes traceability difficult.

From now on please create separate PRs for different changes. Also, try keep the PR subject up to date if (only for a good reason) the PR scope changes.

File: site/features.md (hunk 2 above)
Comment (abuehrle) [inline site/features.md]: I will add a note about the logs terminals as well.

Comment (2opremio) [inline site/features.md]: My point is that there isn't a single terminal, you can open many.

File: site/features.md [hunk 3]
```diff
@@ -63,8 +63,9 @@ Choose an overview of your container infrastructure, or focus on a specific micr
 
 ##<a name="interact-with-and-manage-containers"></a>Troubleshoot and Manage Containers
 
-Troubleshoot your app by directly interacting with containers: pause, restart and stop containers without having to leave the Scope browser window.
-A convenient terminal window is provided that enables you to interact with your app and to troubleshoot and diagnose any issues all within the same context.Select `<_` to launch a terminal in any container from the details panel. 
+Click on a container, pod or host to view the controls that allow you to: pause, restart, stop and delete without having to leave the Scope browser window. Logs of selected containers or pods (if you are running in Kubernetes) can also be viewed by clicking the terminal icon.
+
+And if further troubleshooting is required, a convenient terminal window can also be launched from any container that enables you to interact with your app and run any UNIX command to diagnose issues.  Launch a terminal by clicking the `<_` icon from the details panel of a selected container.
```
Comment (2opremio) [inline site/features.md]: The icon is `>_` not `<_`

**Comment by 2opremio** [comment]
> Thinking about putting all of the search criterion on its own page as reference as well, wdyt? 

OK by me

File: site/features.md [hunk 4]
```diff
@@ -6,44 +6,70 @@ menu_order: 15
...
+Scope builds logical topologies of your application and infrastructure.  A topology is a collection of nodes and edges, where nodes represent objects like processes, container or hosts.  In Scope, edges indicate TCP connections between nodes.  Different node types can be filtered on and viewed and specific microservices can be drilled down on and inspected. Nodes are presented in a particular order with the clients above servers. As a general rule, you can read a Weave Scope view by going from top to bottom.
+
+!['Automatic Topology Mapping in Graph Mode'](images/topology-map.png)
+
+##<a name="views"></a>Views in Scope
+
+Views in Scope can be thought of as high-level filters on your containerized microservices as they run in the cloud. They are categorized into: Processes, Containers, Orchestrators, and Hosts. The Orchestrator view is context-sensitive, and as a result, if an app is running in Kubernetes, then Scope displays Pods, Replica Sets, Deployments, and Services that allow you to drilldown on Kubernetes clusters. 
+
+!['ECS Views in Scope'](images/kubernetes-view.png)
```
Comment (2opremio) [inline site/features.md]: Tooltip is wrong, it should be `Kubernetes in Scope`

File: site/features.md [hunk 5]
```diff
@@ -6,44 +6,70 @@ menu_order: 15
...
+
+If your app is deployed to the Amazon EC2 cluster service, then Scope displays Services and Tasks that make up an ECS cluster.
+
+!['ECS Views in Scope'](images/ECS-view-tasks.png)
+
+For information on how to install Scope on Kubernetes see, "[Installing Scope on Kubernetes](installing.md#k8s)
+
+For information on how to install Scope on the AWS EC2 Container Service see, "[Installing Scope on Amazon ECS](installing.md#ecs)
+
+Under the hosts view, and if you are running Weave Net for container networking a specific Weave Net view appears. This view is useful for troubleshooting any networking problems you may be having. This view displays a number of Weave Net specific attributes such as whether quorum has been reached, the IP addresses used, whether fast datapath is enabled, or if encryption is running and many other useful attributes. See [Weave Net User Guide](https://www.weave.works/docs/net/latest/features/) for more information.
```
Comment (2opremio) [inline site/features.md]: missing comma after networking _and if you are running Weave Net for container networking, a specific Weave Net view appears_

File: site/features.md [hunk 6]
```diff
@@ -6,44 +6,70 @@ menu_order: 15
...
+!['CPU usage indicator in Weave Scope'](images/hosts-cpu.png)
 
-###<a name="powerful-search"></a> Powerful Search
+##<a name="powerful-search"></a> Powerful Search
 
-Powerful search capability enables you to find node types, containers and processes by name, label or even path.  The search functionality supports simple operands so that you can for example, find processes consuming a certain Memory limit or nodes using too much CPU. You can also stack filters to create custom, complex search criterion. 
+Powerful search capability enables you to find node types, containers and processes by name, label or even path.  The search functionality supports simple operands so that you can for example, find processes consuming a certain Memory limit or nodes using too much CPU. Search terms may also be stacked to create custom, complex search criterion. See the online help for a full list of allowable operands.
```
Comment (2opremio) [inline site/features.md]: **The** Powerful search capability?

Comment (2opremio) [inline site/features.md]: _consuming a certain memory limit_ sounds odd

_consuming more than certain memory_ sounds better

Comment (2opremio) [inline site/features.md]: By _online help_ you mean in the app itself?

* If yes, please try to make it more clear
* If not, please include a link to the documemtation you are referring to.

File: site/features.md [hunk 7]
```diff
@@ -6,44 +6,70 @@ menu_order: 15
...
 
-!['Stack Filters'](images/search-python.png)
+!['Powerful Search'](images/search.png)
 
 ##<a name="real-time-app-and-container-metrics"></a>Real-time App and Container Metrics
 
-View contextual metrics, tags and metadata for your containers.  Effortlessly navigate between processes inside your container to the hosts that your containers run on, arranged in expandable, sortable tables.
+View contextual metrics, tags and metadata for your containers by clicking on a node to display its details panel. Drilldown on processes inside your container to the hosts that your containers run on, arranged in expandable, sortable tables.
```
Comment (2opremio) [inline site/features.md]: Shouldn't it be _Drill down_? (note the space)

File: site/features.md [hunk 8]
```diff
@@ -6,44 +6,70 @@ menu_order: 15
...
 
 Choose an overview of your container infrastructure, or focus on a specific microservice. Identify and correct issues to ensure the stability and performance of your containerized applications.
 
-##<a name="interact-with-and-manage-containers"></a>Interact With and Manage Containers
+##<a name="interact-with-and-manage-containers"></a>Troubleshoot and Manage Containers
 
-Interact with your containers directly: pause, restart and stop containers without having to leave the Scope browser window.
+Click on a container, pod or host to view the controls that allow you to: pause, restart, stop and delete without having to leave the Scope browser window. Logs of selected containers or pods (if you are running in Kubernetes) can also be viewed by clicking the terminal icon.
```
Comment (2opremio) [inline site/features.md]: I would place _(if you are running in Kubernetes)_ after the first mention of pod

File: site/features.md [hunk 9]
```diff
@@ -6,44 +6,70 @@ menu_order: 15
...
 
-##<a name="troubleshoot-apps"></a>Troubleshoot Apps
+And if further troubleshooting is required, terminal windows can be launched from any container so that you can interact with your app and run any UNIX command to diagnose issues.  Launch a terminal by clicking the `>_` icon from the details panel of a selected container.
```
Comment (2opremio) [inline site/features.md]: They can also be launched from hosts

File: site/features.md (hunk 7 above)
Comment (abuehrle) [inline site/features.md]: No, in this case it's one word.

**Comment by abuehrle** [comment]
I am finished with these changes for now.  If I do more work on the scope docs, I'll create a new PR.

File: site/features.md [hunk 10]
```diff
@@ -63,9 +63,9 @@ Choose an overview of your container infrastructure, or focus on a specific micr
...
 
-Click on a container, pod or host to view the controls that allow you to: pause, restart, stop and delete without having to leave the Scope browser window. Logs of selected containers or pods (if you are running in Kubernetes) can also be viewed by clicking the terminal icon.
+Click on a container, pod or host to view the controls that allow you to: pause, restart, stop and delete without having to leave the Scope browser window. Logs of selected containers or pods (if you are running Kubernetes) can also be displayed by clicking the terminal icon.
 
-And if further troubleshooting is required, terminal windows can be launched from any container so that you can interact with your app and run any UNIX command to diagnose issues.  Launch a terminal by clicking the `>_` icon from the details panel of a selected container. 
+And if further troubleshooting is required, terminal windows can be launched from any container or host so that you can interact with your app and run any UNIX command to diagnose issues.  Launch a terminal by clicking the `>_` icon from the details panel of a selected container.
```
Comment (2opremio) [inline site/features.md]: _of a selected **host** or container._

Comment (abuehrle) [inline site/features.md]: What what I wrote is correct: ... terminal windows can be launched from any container or host

Comment (2opremio) [inline site/features.md]: But then you only refer to containers at the end.

Comment (2opremio) [inline site/features.md]: Which is correct but incomplete

Comment (abuehrle) [inline site/features.md]: I like how all of my stuff get extra-special scrutiny.  Have you finished reviewing that other open doc request yet? 

Or I guess this on is perfect: 

https://github.com/weaveworks/scope/pull/2257

**Comment by 2opremio** [comment]
Yes, I did finish, see
https://github.com/weaveworks/scope/pull/2257#issuecomment-280660606

**Comment by 2opremio** [comment]
Ping

**Pull request merged**